modin/pandas/test/dataframe/test_binary.py

```diff
@@ -353,3 +354,17 @@ def test_add_custom_class():
         *create_test_dfs(test_data["int_data"]),
         lambda df: df + CustomIntegerForAddition(4),
     )
+
+
+def test_non_commutative_multiply():
+    # This test checks that mul and rmul do different things when
+    # multiplication is not commutative, e.g. for adding a string to a string.
```

Comment:
```suggestion
    # multiplication is not commutative, e.g. for multiplying a string with an integer.
```
feels weird to provide an example of adding for multiplication operation 😉